soh/src/overlays/actors/ovl_Obj_Switch/z_obj_switch.c

```diff
@@ -295,6 +295,14 @@ void ObjSwitch_Init(Actor* thisx, PlayState* play) {
 
     if (type == OBJSWITCH_TYPE_FLOOR || type == OBJSWITCH_TYPE_FLOOR_RUSTY) {
         this->dyna.actor.world.pos.y = this->dyna.actor.home.pos.y + 1.0f;
+        if (CVarGetInteger("gEnhancements.FixFloorSwitches", 0)) {
+            if (this->dyna.actor.params == 4608) {
+                this->dyna.actor.world.pos.y = -0x30B;
+            }
+            if (this->dyna.actor.params == 14848) {
+                this->dyna.actor.world.pos.y = 0x1194;
+            }
```

Comment:
You only need to move the switches down by 1 unit correct? If so, you could instead just write the following to avoid hardcoded values:

```suggestion
            if (this->dyna.actor.params == 4608 || this->dyna.actor.params == 14848) {
                this->dyna.actor.world.pos.y -= 1;
            }
```

Reply:
Ah, yes, that is a much better way. Didn't think of that